.eslintrc

```diff
@@ -129,7 +146,7 @@
     "no-undef": 2,                   // disallow use of undeclared variables unless mentioned in a /*global */ block
     "no-undefined": 0,               // disallow use of undefined variable (off by default)
     "no-undef-init": 1,              // disallow use of undefined when initializing variables
-    "no-unused-vars": [1, {"vars": "all", "args": "none"}], // disallow declaration of variables that are not used in the code
+    // "no-unused-vars": [1, {"vars": "all", "args": "none"}], // disallow declaration of variables that are not used in the code
```

Comment:
Good to know! However, it looks like this still shouldn't be disabled. The jsx-uses-vars docs say:

```
This rule has no effect if the no-unused-vars rule is not enabled.
```